pilot/pkg/features/pilot.go

```diff
@@ -448,8 +448,9 @@ var (
 	InjectionWebhookConfigName = env.Register("INJECTION_WEBHOOK_CONFIG_NAME", "istio-sidecar-injector",
 		"Name of the mutatingwebhookconfiguration to patch, if istioctl is not used.").Get()
 
-	ValidationWebhookConfigName = env.Register("VALIDATION_WEBHOOK_CONFIG_NAME", "istio-istio-system",
-		"Name of the validatingwebhookconfiguration to patch. Empty will skip using cluster admin to patch.").Get()
+	EnableValidatingWebhookConfigPatch = env.Register("PILOT_ENABLE_VALIDATING_WEBHOOK_CONFIG_PATCH", true,
```

Comment:
This is a breaking change, can we keep the old one? and just update the docs.

Reply:
i think maybe, all changes have been reverted, and only upate the docs.